udata/core/organization/commands.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+
+from udata.commands import manager
```

Comment:
better use a submanager for namespacing on organization.
See: https://github.com/etalab/udata/blob/master/udata/harvest/commands.py#L10-L17